Clinical Key.js

```diff
@@ -0,0 +1,190 @@
+{
+	"translatorID": "a55463ba-e403-415b-80d4-284d5f9b4b15",
+	"label": "Clinical Key",
+	"creator": "Jaret M. Karnuta",
+	"target": "^https://(www\\.|www-)clinicalkey(\\.|-)com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-12-26 21:15:27"
+}
+
+/*
+This translator is designed specifically for use on book section portions of
+ClinicalKey. It will not work on journal articles and book overview pages.
+
+NB: url and doc.location.href are different. I think it is because of the way clinicalkey redirects.
+Replicate by going to a content page (/content/book/...) and then going to a broswing page (/browse/book/...).
+URL remains the page of the previous page (content page) and doc.location.href is the current page (as it should be).
+
+Hence, I never use url and change its content to doc.location.href in detectWeb, the only function that uses the url
+*/
+
+function detectWeb(doc, url) {
+	//see NB above for explanation
+	url = doc.location.href;
+
+	//contentType depends on url, which is present, but rest of site is loaded via ajax (I think)
+	//monitor dom and reset if changes
+	var jsession = doc.getElementById('jsessionid');
+	Z.debug(jsession.value);
+	if(jsession){
+		Zotero.monitorDOMChanges(jsession, {attributes:true});
+		if(!jsession.value){
+			return;
+		}
+	}
+
+	var contentType;
+	if(url.includes("/content/book/") && !url.includes("login?")){
```

Comment:
I have not seen `includes` before and always use `indexOf` for such cases. IMO cross-browser code is more important than performance (or shorter code).